test-runner-frontend/src/Components/TestRunsTable.test.js

```diff
@@ -79,17 +92,37 @@ describe('Test Runs Table', () => {
   test('Renders sortDate', async () => {
     var testRunResults = generateMockDataCaller(25);
     var sortDate = jest.fn();
-    const { debug, getByTestId } = render(<TestRunsTable testRunResults={testRunResults} sortDate={sortDate} />);
+    const { debug, getByTestId } = render(
+      <BrowserRouter>
+        <TestRunsTable testRunResults={testRunResults} sortDate={sortDate} />
+      </BrowserRouter>
+    );
     const SortButton = screen.getByTestId('iconButton');
     await userEvent.click(SortButton);
     expect(sortDate).toHaveBeenCalled();
   });
 
   test('Renders checkboxActive', async () => {
     var onCheckboxChange = jest.fn();
-    var testRunResults = [generateMockData()];
+    var testRunResults = [
+      {
+        id: crypto.randomUUID(),
+        branch: 'master',
+        status: 'FINISHED',
+        benchmarks: 11,
+        improvedVsRegressed: {
+          improved: 9,
+          regressed: 2,
+        },
+        machine: 'core_32',
+        date: '12/08/2022 4:35 pm',
+      },
```

Comment:
whats the reason for no longer needing to use `generateMockData()`?

Reply:
The reason for this is because in generateMockData() the status field is generated randomly from branchArray but for this test we need it defined as FINISHED.